### DISCUSSION TITLE
Home Assistant data from Predbat plan?

### DISCUSSION THREAD

**Discussion by matknowles** [springfall2008/batpred#872] [Q&A] [answered]
Hello everyone, is it possible to create a home assistant sensor from the predbat plan? I'm specifically interested in getting the cost/profit at the end of the day figure, but it might be useful to extract/use other data from it as well.

Thanks

**Comment by Nilogax** [comment]
Maybe take a look at #807 which was related to daily cost

**Reply by matknowles** [reply]
I didn't see that, thanks for linking it. I've tried to set it up, but it seems like the data is different and not what I'm looking for. My plan currently shows end of day total as being £1.74 (as of now 21:23), but predbat.cost_today on the chart and from that thread shows £1.31, which is my current actual cost as of now, not at the end of the day. I guess I'm looking for the _predicted_ cost at end of day?

**Comment by Nilogax** [comment, marked as answer]
predbat.best_metric contains a list of predicted costs in the attributes so it you want a forward looking value you can get it from there.

**Reply by matknowles** [reply]
That's great, thanks for your help.
